### DISCUSSION TITLE
Vertical option for Command Bar

### DISCUSSION THREAD

**Discussion by konero** [opentoonz/opentoonz#6485] [Ideas]
**Is your feature request related to a problem? Please describe**
Downsized my drawing display to 16 inch, there's not a lot of room in terms of height.

**Describe the solution you'd like**
To add a vertical orientation option to the Command Bar like the Tool Bar, so it can be docked to the side.

**Comment by artisteacher** [comment]
I'd love to see a more responsive option. Where the commands would wrap to fit whatever amount of space is in the panel. If the panel was made narrow enough, it would automatically work for a vertical orientation.

**Comment by DarrenTAnims** [comment]
Perhaps the orientation would change to a horizontal bar when you try to dock it above or below a panel, and to a vertical bar when docking to the left or right of a panel.

**Comment by RodneyBaker** [comment]
Do the new custom panels address this request or is the full focus on current command bar only?

**Comment by konero** [comment]
@RodneyBaker The new custom panels are really good, though there's not a template for many tool buttons compared to the command bar. I don't have Qt Designer setup, so I haven't gotten around to looking at templates.

Some templates like...

16 rows 1 col
16 rows 2 cols
16 rows 3 cols

...would be neat, and probably solve this issue.

**Comment by RodneyBaker** [comment]
@konero 
@shun-iwasawa posted a link in the Custom Panels PR that only downloads Qt Designer.
It's a really fast download and really easy to get going on custom panels... just import a current .ui file as an example as that will get you up to speed really fast.

It took me less than 15 minutes from clicking the link to modifying one of Shun's custom panels into a 16x16 grid of buttons.

Link to (tiny download of only) QtDesigner:  https://build-system.fman.io/qt-designer-download

Link to Custom Panels PR (for information):  https://github.com/opentoonz/opentoonz/pull/4495

There should be two example .ui files/panels in the template directory in stuff/libraries.

I am confident you will be quite dangerous with these new custom panels!  :)
